aiger/bv.py

```diff
@@ -1,23 +1,13 @@
 from aiger import common, parser
 
+import string
+import re
 
-def _bv_andor(wordlen, gate, output='x&y', left='x', right='y'):
-    aig = common.empty()
-    for i in range(wordlen):
-        aig |= common.and_gate(
-            [f'{left}[{i}]', f'{right}[{i}]'], output=f'{output}[{i}]')
-    return aig
-
+forbidden = [' ', '[', ']', '\n', '\t', '\r', '\x0b', '\x0c']
+var_name_alphabet = set([c for c in string.printable if c not in forbidden])
```

Comment:
Please do all caps for constants and change to immutable types. In this case, list goes to tuple and set goes to frozenset.